src/java.base/share/classes/com/sun/crypto/provider/DHPublicKey.java

```diff
@@ -265,10 +267,10 @@ public String toString() {
         return sb.toString();
     }
 
-    private void parseKeyBits() throws InvalidKeyException {
```

Comment:
Same comment about moving this code block to constructor since it's only used once and the small size.